### ISSUE TITLE
Session ID not found. immediately after connection attempt

### ISSUE THREAD

**Issue by Tealk** [lucent-sea/Remotely#270]
Hello,

I installed my own instance today and was about to test it, but as soon as I try to connect I get the message "Session ID not found".
I installed the whole thing on an empty Ubuntu 20.04.2 LTS with the "Ubuntu_Server_Install.sh" sc

**Comment by opensource008** [comment]
not sure but if not hardcoded the url you need to manual supply the url to remotely quick support click the gear icon..

**Comment by Tealk** [comment]
is this also the case when i download remote quck support from my own instance?

**Comment by opensource008** [comment]
yes if you build your own remotely is already hardcoded the url on quick support and agent installer

**Comment by lucent-sea** [comment]
This could also be due to Nginx misconfiguration.

I have a preview release out that has an entirely different build process.  Check out the notes for the preview build on the releases page and go over the build instructions on the README again.  :) 

Also, Caddy is the new default web server.  I'd recommend using it for fresh deployments.

**Comment by opensource008** [comment]
> This could also be due to Nginx misconfiguration.
> 
> I have a preview release out that has an entirely different build process. Check out the notes for the preview build on the releases page and go over the build instructions on the README again. :)
> 
> Also, Caddy is the new default web server. I'd recommend using it for fresh deployments.

hopping there is still installation script for linux that default for nginx configuration.. thanks and more power sir @lucent-sea

**Comment by lucent-sea** [comment]
> > This could also be due to Nginx misconfiguration.
> > I have a preview release out that has an entirely different build process. Check out the notes for the preview build on the releases page and go over the build instructions on the README again. :)
> > Also, Caddy is the new default web server. I'd recommend using it for fresh deployments.
> 
> hopping there is still installation script for linux that default for nginx configuration.. thanks and more power sir @lucent-sea

Yup! The new Server Installer tool let's you select Nginx or Caddy.

**Comment by Tealk** [comment]
I ran the the installation script on 13, there was still installed nginx; then i should probably wait with my test setup ?

**Comment by opensource008** [comment]
> > > This could also be due to Nginx misconfiguration.
> > > I have a preview release out that has an entirely different build process. Check out the notes for the preview build on the releases page and go over the build instructions on the README again. :)
> > > Also, Caddy is the new default web server. I'd recommend using it for fresh deployments.
> > 
> > 
> > hopping there is still installation script for linux that default for nginx configuration.. thanks and more power sir @lucent-sea
> 
> Yup! The new Server Installer tool let's you select Nginx or Caddy.

sorry to mix this issue to this post, happy to know for nginx,, as of now im more convinced to use nginx.

**Comment by opensource008** [comment]
> I ran the the installation script on yesterday 13. and there was still installed nginx; then i should probably wait with my test setup ?

if you use the script from latest realese yes still use the default nginx, the new one sir jared mentioned is still on dev.

**Comment by Tealk** [comment]
but then the statement in the readme is a little confusing
![image](https://user-images.githubusercontent.com/12276250/114823909-6ba41b80-9dc4-11eb-9ba4-d00fa956f3c2.png)
